.pre-commit-config.yaml

```diff
@@ -11,7 +11,13 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
       - id: check-merge-conflict
-  # NOTE: Keep order as pycln (remove unused imports), then isort (sort them) and black (final formatting)
+  # NOTE: Keep order as pyupgrade (will update code) then pycln (remove unused imports), then isort (sort them) and black (final formatting)
```

Comment:
if you pass `args: ["--profile", "black"]` to isort it won't conflict with black